io/src/main/scala/sbt/io/IOSyntax.scala

```diff
@@ -0,0 +1,13 @@
+package sbt.io
+
+trait IOSyntax extends PathExtra /* with URIExtra */ {
+  type File = java.io.File
+  type URI = java.net.URI
+  type URL = java.net.URL
+
+  def uri(s: String): URI = new URI(s)
+  def file(s: String): File = new File(s)
+  def url(s: String): URL = new URL(s)
+}
+
+object syntax extends IOSyntax
```

Comment:
@dwijnand I half understand this, but we're definitely asking for heartache mixing traits + package objects....